packages/fms/package.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Fms(CMakePackage):
+    """GFDL's Flexible Modeling System (FMS) is a software environment
+    that supports the efficient development, construction, execution,
+    and scientific interpretation of atmospheric, oceanic, and climate
+    system models."""
+
+    homepage = "https://github.com/NOAA-GFDL/FMS"
+    url = "https://github.com/NOAA-GFDL/FMS/archive/refs/tags/2022.04.tar.gz"
```

Comment:
Hopefully we can implement something like `url_for_version()` in the future instead of hardcording this. e.g. This is what we do in the MOM5 SPD:
```
    def url_for_version(self, version):
        return "https://github.com/ACCESS-NRI/mom5/tarball/{0}".format(version)
```

Reply:
No longer relevant because there is no `fms` package in the repo.